### ISSUE TITLE
"skip to navigation" link has no valid target

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.theme#32]
in the home page document we find:

 `<a accesskey="6" href="http://isaw4-dev.atlantides.org/#portal-globalnav">Skip to navigation</a>`

but there is no element in the page with `id="portal-globalnav"`.

This breaks function for users of text-only browsers (i.e., those with Braille displays) and voice browsing.

A similar problem occurs on all other pages.

**Comment by skleinfeldt** [comment]
This is probably in @alecpm 's court.

**Comment by alecpm** [comment]
Do we want to keep the skip links and update them or eliminate them?  In this case the navigation is at the top of the page, so it's perhaps not so useful (though maybe in a text browser it is).

**Comment by paregorios** [comment]
I'm happy to get rid of "skip to navigation" in our case because it is at the top. But "skip to content" (which I think does work) should be kept.

**Comment by paregorios** [comment]
as much as it pains me to defer an accessibility issue, I'm putting this on the backlog

**Comment by paregorios** [comment]
Transferred to https://github.com/isawnyu/isaw.web/issues/69
